silo-core/contracts/incentives/interfaces/ISiloIncentivesController.sol

```diff
@@ -0,0 +1,221 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.8.28;
+
+import {IDistributionManager} from "./IDistributionManager.sol";
+import {DistributionTypes} from "../lib/DistributionTypes.sol";
+
+interface ISiloIncentivesController is IDistributionManager {
+    event RewardsAccrued(address indexed user, address indexed rewardToken, bytes32 indexed programId, uint256 amount);
+    event ClaimerSet(address indexed user, address indexed claimer);
+    event IncentivesProgramCreated(bytes32 indexed programId, string indexed name);
+    event IncentivesProgramUpdated(bytes32 indexed programId);
```

Comment:
```suggestion
    event IncentivesProgramUpdated(string indexed programId);
```

I would use string everywhere because it is much easier to convert string to bytes32 than bytes32 -> string.